src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,30 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+
     public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = new ManufacturerDaoImpl();
+        Manufacturer manufacturer1 = new Manufacturer(null, "Lincoln", "USA");
+        Manufacturer manufacturer2 = new Manufacturer(null, "Ford", "USA");
+        Manufacturer manufacturer3 = new Manufacturer(null, "Audi", "Germany");
+        Manufacturer manufacturer4 = new Manufacturer(null, "BMW", "Germany");
+        manufacturerDao.create(manufacturer1);
+        manufacturerDao.create(manufacturer2);
+        manufacturerDao.create(manufacturer3);
+        manufacturerDao.create(manufacturer4);
```

Comment:
This code has redundant variables, instead of creating object Manufacturer manufacturer1 = new Manufacturer(null, "Lincoln", "USA"); and then manufacturerDao.create(manufacturer1); u can refactor it like that:
manufacturerDao.create(new Manufacturer(null, "Lincoln", "USA")) and why u are calling constructor(id, name, country) and pass id - null, u have a constructor (name, country) u can call it here. Also, u can create an array of Manufactures and after that in the for-cycle call create method.
When u creating variable names, manufacturer1 manufacturer2 manufacturer is not good, u can call it: lincoln,for,audi, using data from fields from an object.